pkg/builtin/http/http.go

```diff
@@ -128,9 +128,11 @@ func (c *HTTPCmd) Run(meta *registry.Meta) (res interface{}, err error) {
 	b, err := io.ReadAll(resp.Body)
 	// parse response body and headers
 	return map[string]interface{}{
-		"body":    string(b),
-		"header":  resp.Header,
-		"trailer": resp.Trailer,
+		"body":        string(b),
+		"header":      resp.Header,
+		"trailer":     resp.Trailer,
+		"status":      resp.Status,
+		"status_code": resp.StatusCode,
```

Comment:
```suggestion
		"statusCode": resp.StatusCode,
```

Reply:
status seems not necessary

Reply:
Does status contain error reason messages when error occurred ?

Reply:
no, it won't. The useful information contained in the response body.

Reply:
ok, i'll fix it.